CMakeLists.txt

```diff
@@ -123,6 +123,15 @@ if (ENABLE_COVERAGE AND CMAKE_COMPILER_IS_GNUCXX)
     setup_target_for_coverage(${PROJECT_NAME}_coverage ctest coverage "-j2;--output-on-failure")
 endif()
 
+find_package(libmongocxx REQUIRED)
+find_package(libbsoncxx REQUIRED)
+include_directories(${LIBMONGOCXX_INCLUDE_DIR})
+include_directories(${LIBBSONCXX_INCLUDE_DIR})
+include_directories("/usr/local/include/mongocxx/v_noabi")
+include_directories("/usr/local/include/bsoncxx/v_noabi")
+include_directories("/usr/local/include/libmongoc-1.0")
+include_directories("/usr/local/include/libbson-1.0")
+include_directories("/usr/local/lib")
```

Comment:
Need some clean-ups on these `include_directories` commands. Try to keep it minimum.

Reply:
I dont know why but does not work without this

Reply:
I think the find_package helps you to include some libs and you can remove those from "include".